lib/scan.ts

```diff
@@ -29,7 +31,12 @@ export async function scan(options: Options): Promise<PluginResponse> {
     );
 
     const filteredSignatures: SignatureResult[] = allSignatures.filter((s) => {
-      return s !== null;
+      if (s === null) {
```

Comment:
I'm not totally satisfied with doing this inside the filter-call. I have tested different solution, and still this is the best one so for.

One of the other alternatives was `map`. Maybe misleading as well, so I ended up using the filter, since it removes one "loop" from the implementation.